barretenberg/cpp/src/barretenberg/flavor/relation_definitions.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "barretenberg/relations/relation_types.hpp"
+
+// Utilities for explicit instantiation of relation and permutation classes.
```

Comment:
I should have git mv'd this file, but this just removes the parts with 'extern', simplifying a bit, and the name _fwd.hpp no longer made sense.